templates/hotgraphicLayoutPins.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { classes } from 'core/js/reactHelpers';
```

Comment:
additional import for compile 

`import { classes, compile } from 'core/js/reactHelpers';`